activities/codebase-stewardship/assessment-issue-template.md

```diff
@@ -0,0 +1,24 @@
+---
+type: Resource
+---
+
+# Template for assessment issues
+
+The text below can be used as a template for creating an issue in a repository for codebase where we are [starting an assessment](open-assessment.md). For easiest copy-paste it is rendered with code style here.
+
+## Template text
+
+`**Purpose**`
+
+`This issue helps tracking the assessment of this repository on it's way to [stewardship](https://about.publiccode.net/activities/codebase-stewardship/) by the Foundation for Public Code. The assessment is compiled into this [spreadsheet](linkToSpreadsheet).`
+
+`**Steps**`
+
+`- [ ] Business case`\
+`- [ ] Co-develop-ability`\
+`- [ ] Community`\
+`- [ ] Community potential`\
+`- [ ] Code reusability`\
+`- [ ] Reuse potential`\
+`- [ ] Societal impact`\
+`- [ ] Story`
```

Comment:
This can be a [fenced code block](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code)

Reply:
Indeed, I will change it!